kie-wb/kie-wb-webapp/src/main/java/org/kie/workbench/backend/DashbuilderBootstrap.java

```diff
@@ -47,17 +48,31 @@
     @Inject
     protected DataSetDefRegistry dataSetDefRegistry;
 
+    @Inject
+    protected EntityManagerFactory emf;
+
     @PostConstruct
     protected void init() {
+        // figure out data source JNDI name
+        Object ds = emf.getProperties().get("hibernate.connection.datasource");
+        if (ds != null && ds instanceof javax.naming.Referenceable) {
+            try {
+                jbpmDatasource = ((javax.naming.Referenceable) ds).getReference().getClassName();
+            } catch (Exception e) {
```

Comment:
if it fails then it means it does not implement that interface and thus will fallback to this settings:
https://github.com/droolsjbpm/kie-wb-distributions/pull/85/files#diff-69c5ff9ccbae9219ce6f30a08dd82aecR33
either hardcoded or given as system property